nixos/lib/testing-python.nix

```diff
@@ -136,25 +136,26 @@ in rec {
       # interactively with the specified network, and for starting the
       # VMs from the command line.
       driver = let warn = if skipLint then lib.warn "Linting is disabled!" else lib.id; in warn (runCommand testDriverName
-        { buildInputs = [ makeWrapper];
-          testScript = testScript';
-          preferLocalBuild = true;
-          testName = name;
-        }
+      {
+        buildInputs = [ makeWrapper ];
+        testScript = testScript';
+        preferLocalBuild = true;
+        testName = name;
+      }
```

Comment:
1. body `in` doesn't get expanded, which cause the attribut unindented.
2. the multiline string shouldn't be indented?

Reply:
solved by indenting inline lambda function.